.github/workflows/install.yaml

```diff
@@ -10,9 +10,9 @@ jobs:
       matrix:
         python-version:
           - 2.7
-          - 3.6
```

Comment:
Why removing 3.6 and 3.7?

Reply:
It's somewhat of an arbitrary choice because 3.9 and 3.10 were added. Feel free to leave them in.

Reply:
I can't update your branch, I'd have to create my own PR, your call.